src/components/calculator-page.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Calculator from './Calculator';
+import '../App.css';
+
+const CalculatorPage = () => (
+  <section className="calc">
+    <h2>Let&apos;s do Magic!</h2>
+    <Calculator />
+  </section>
+);
+
+export default CalculatorPage;
```

Comment:
- [ ] Nice one implementing your own design 🔥. However, let's try to match the general layout of the calculator's page to the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). This is to ensure that you have gotten the skill to replicate an app from a given wireframe. So let's move the `<h2>Let&apos;s do Magic!</h2>` to the left side of the page and the `calculator` itself to the right.  💪

Reply:
Thank you for that, but I'm gonna try one more time without this one, it really feels weird on the left.